lib/Dialect/XeTile/IR/XeTileOps.cpp

```diff
@@ -351,7 +351,8 @@ bool verifyInnerBlocksWithVecShape(mlir::DenseI32ArrayAttr &innerBlocks,
 mlir::LogicalResult LoadTileOp::verify() {
   auto encoding = getSource().getType().getEncoding();
   auto tileShape = getSource().getType().getShape();
-  if (!encoding)
+  // FIXME: TileAttr innerBlocks is not being setup properly by tiling pass
```

Comment:
what would be the fix for this? I believe tiling pass needs to  attach this info into tile data type?